src/models/business.js

```diff
@@ -0,0 +1,25 @@
+export default (sequelize, DataTypes) => {
+  const Business = sequelize.define('business', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    fantasyName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    }
+  })
+
+  Business.associate = (models) => {
+    Business.belongsTo(models.legalInformation)
+    Business.hasOne(models.user)
+    Business.belongsToMany(models.physicalAddress, {
```

Comment:
Maybe there is a requirement I don't know about, but this doesn't make much sense to me. I'd have written this as "Business.hasMany(models.physicalAddress)".